test/test_integration.rb

```diff
@@ -240,6 +240,7 @@ def test_restart_closes_keepalive_sockets_workers
   end
 
   def test_restart_with_prune_bundler_keeps_bundler_env
+    skip NO_FORK_MSG unless HAS_FORK
     initial_reply, new_reply = restart_server_and_listen("-q -w 2 --prune-bundler test/rackup/hello-bundler-env.ru", "PATH" => "hello-bundler:#{ENV["PATH"]}", "BUNDLER_ORIG_PATH" => nil)
```

Comment:
as in do you actually need "-w 2" here or not.

Reply:
The `-w 2` should not be needed, but I cannot get the build to succeed on AppVeyor. The errors seem to be compile errors? Any tips here would be appreciated! 🙏

Reply:
At present, Puma isn't installed with bundler on Appveyor.  Sometime soon, but probably after a determination about using GitHub Actions for CI...